persistence/state.go

```diff
@@ -354,17 +354,18 @@ func (p *PostgresContext) updateTransactionsTree() error {
 }
 
 func (p *PostgresContext) updateParamsTree() error {
-	paramsOrFlags, err := p.getParamsOrFlags(types.ParamsTableName, p.Height, false)
+	params, err := p.getParamsUpdated(p.Height)
 	if err != nil {
 		return err
 	}
 
-	for _, pf := range paramsOrFlags {
-		pfHash, err := pf.Hash()
+	for _, param := range params {
+		paramHash := crypto.SHA3Hash([]byte(param.String()))
```

Comment:
s/paramHash/paramKey

This is the key used to index the state of that param.

`Merkle Trees` -> A data structure that can provide a succient argument (the root) of our entire state
`Key-Value Store` -> A data store engine backing the Merkle Tree. The path to every leaf is the key, and the value is the data.

Ditto for flag